project_event/models/project_task.py

```
@@ -282,11 +282,50 @@ def create_new_activity(self, vals):
             .next_by_code('project.task.activity')
         return super(Task, self).create(vals)
 
+    @api.multi
+    def get_partners(self):
+        partners = []
+        for e in self.employee_ids:
+            if e.user_id:
+                partners.append(e.user_id.partner_id.id)
+            else:
+                raise UserError(
```

@lgharib the raise error is not handled when updating task employee_ids

---

True